src/components/SiteInformation/SiteInformation.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert, TextLink } from '@/component-library';
+
+const SiteInformation = (): JSX.Element => {
+  const hasLink = !!process.env.REACT_APP_SITE_INFORMATION_LINK;
+
+  return (
+    <Alert status='warning'>
+      {process.env.REACT_APP_SITE_INFORMATION_MESSAGE}
+      {hasLink && (
+        <>
+          {' '}
+          <TextLink external icon to={process.env.REACT_APP_SITE_INFORMATION_LINK || ''}>
```

Comment:
I've had to use the OR operator here because even with the `hasLink` variable—which will prevent this component rendering if there is no link value—there was a type checking error (`Type 'string | undefined' is not assignable to type 'LocationDescriptor<unknown>`).